source/MRViewer/MRRibbonButtonDrawer.cpp

```diff
@@ -402,10 +407,22 @@ bool RibbonButtonDrawer::GradientRadioButton( const char* label, int* value, int
     return res;
 }
 
+bool RibbonButtonDrawer::GradientColorEdit4( const char* label, Vector4f& color, ImGuiColorEditFlags flags /*= ImGuiColorEditFlags_None */ )
+{
+    ImVec2 framePadding( 8.f, 3.f );
+    ImGui::PushStyleVar( ImGuiStyleVar_FramePadding, framePadding );
+
+    const bool changed = ImGui::ColorEdit4( label, &color.x, flags );
+    ImGui::PopStyleVar();
+    return changed;
+}
```

Comment:
Why is called Gradient (is it supposed to be make such in future, if so please add TODO commet)